### PR TITLE
Update, organize, and annotate README and specification files.

### PR DESCRIPTION
**Pull request by rjb25**
When in doubt use brute force.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
+If you'd like access to this, or have any suggestions or questions just email me at stop-loss-gitters@basanese.com. 
+Also current editor email can be found at shadybear18@gmail.com
 
-It looks like the best way to implement this will be on a basic web server with node.js. However, there's already ruby and java code for working with BTC-E. 
+This program will automate trades on bitreserve.org, Kraken, or anywhere else that allows instant trading in relation to historical prices.
 
-This program will do one thing, and one thing only: Automate trades on BTC-E. 
-
-Thoughts? Send them. 
+It looks like the best way to implement this will be on a basic web server with node.js. houghts? Send them.
```
Comment (v6) [inline README:6]: ... Um, houghts?

File: specification_for_a_buy_sell_bot_for_later_application_of_algorithms.txt [hunk 2]
```diff
@@ -1,28 +1,78 @@
-// Need to talk to Paolo, to see if I'm going in the right direction, here. 
+This program will automate trades on bitreserve.org, Kraken, or anywhere else that allows instant trading in relation to historical prices.
 
-//https://btc-e.com/api/documentation
+Here is an overview of what it does;
```
Comment (v6) [inline specification_for_a_buy_sell_bot_for_later_application_of_algorithms.txt:3]: // , Is this supposed to be a semicolon?

**Comment by v6** [comment]
// , Please review the whole thing to make sure there aren't any errata I have missed.

File: README (hunk 1 above)
Comment (rjb25) [inline README:6]: I re worded it

**Pull request closed without merging**